samples/artifacts/DataPipelining/StartupTrigger.siddhi

```diff
@@ -0,0 +1,27 @@
+@App:name("StartupTrigger")
+@App:description("A trigger to generate an event at SiddhiApp startup.")
```

Comment:
generate an event at SiddhiApp startup. Make  sure to use first couple of words in descriptions to give short overview, a this will be displayed to the user prominently